pkg/kubelet/eviction/memory_threshold_notifier_windows.go

```diff
@@ -90,12 +90,12 @@ func (m *windowsMemoryThresholdNotifier) UpdateThreshold(summary *statsapi.Summa
 func (m *windowsMemoryThresholdNotifier) Description() string {
 	var hard, allocatable string
 	if isHardEvictionThreshold(m.threshold) {
-		hard = "hard "
```

Comment:
nit: Can we move these changes to the commit that first introduced them earlier?

Reply:
I'll rebase the changes down to 1 or 2 commits.